post_install.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-python -m pip install "git+https://github.com/AllenNeuralDynamics/aind-ng-link@feat-zarr-checker#egg=aind-ng-link" --no-cache-dir
+python -m pip install "git+https://github.com/AllenNeuralDynamics/aind-ng-link" --no-cache-dir
```

Comment:
Is this published to pypi now? We can probably just add it to the pyproject.toml file

Reply:
I think @dyf had published an older version at some point https://pypi.org/project/aind-ng-link/1.0.6/
Not sure if the latest version is up.

Reply:
Ah no auto-publishing, let me publish the latest version

Reply:
@carshadi Try this out: 
https://pypi.org/project/aind-ng-link/1.0.15/

Reply:
works 👍